### DISCUSSION TITLE
Setting up gluetun in portainer

### DISCUSSION THREAD

**Discussion by cheerfulnut** [qdm12/gluetun#720] [Q&A] [answered]
Hello,

I'm very new to containers/docker so my apologies if I'm missing something very obvious, but my google-fu is failing me..

I've been a Mullvad user for a while and now that I'm trying out docker, I wanted see if could get a VPN container built for other containers to go through.  However, every time I create the container, portainer constantly shows unhealthy status.  The logs show last output as below:

> 2021/11/21 21:50:14 ERROR HTTP response status is not OK: 500 Internal Server Error: lookup github.com on 1.1.1.1:53: write udp 172.20.0.2:37042->1.1.1.1:53: write: operation not permitted 2021/11/21 21:50:14 INFO Shutdown successful

Basically I'm just going into portainer > stacks > add stack > enter the name of the stack (for example "VPN container" and then paste the following configuration into the web editor

> version: "3"
> services:
>   gluetun:
>     image: qmcgaw/gluetun
>     cap_add:
>       - NET_ADMIN
>     ports:
>       - 8888:8888/tcp # HTTP proxy
>       - 8388:8388/tcp # Shadowsocks
>       - 8388:8388/udp # Shadowsocks
>     volumes:
>        - /srv/dockerapps/gluetun:/gluetun
>     environment:
>       # See https://github.com/qdm12/gluetun/wiki
>       - VPNSP=mullvad
>       - VPN_TYPE=wireguard
>       # OpenVPN:
>       #- OPENVPN_USER=
>       #- OPENVPN_PASSWORD=
>       # Wireguard:
>       - WIREGUARD_PRIVATE_KEY=QIWvRxxxxxxxxxo9VGYmU=
>       - WIREGUARD_ADDRESS=1.x.x.x/32
>       - COUNTRY=SINGAPORE
>       # Timezone for accurate log times
>       - TZ=Japan

Would you have any ideas what I'm doing wrong?  Thank you in advance for any pointers...

**Comment by qdm12** [comment, marked as answer]
> I wanted see if could get a VPN container built for other containers to go through

Yes you can definitely do that.

> 500 Internal Server Error

That might be an error on the github side. You can try setting `HEALTH_ADDRESS_TO_PING` to something else like `1.1.1.1` if you keep on getting those errors, and it should then work and not 'auto-heal'.

**Reply by cheerfulnut** [reply]
Thank you!  That fixed the "unhealthy" status!  Much appreciated, now I will continue with figuring out everything else :)
